### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db")
```
Comment (Bohdan-Salamakha) [inline app/managers.py:8]: db should have .sqlite3 extension

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
u need to create all required tables automatically
image

File: app/managers.py (hunk 1 above)
Comment (goldenuni) [reply app/managers.py:8]: @Bohdan-Salamakha 
image
As I understand, I should create DB inside SQLiteStudio and then add it into a project, isn't it? If yes, I've done this, but when I add extension for the file, my program doesn't see the DB. To resolve your problem, I should have added the DB file into Git, but a checklist said, that I [shouldn't](https://github.com/mate-academy/py-actor-manager/blob/master/checklist.md#dont-push-db-files). So what do I need to do?

Comment (ArsenPidhoretskyi) [reply app/managers.py:8]: cinema_db is a file it does not SQL database. Please read about extensions of SQLite
You should not add db file into git. Read about gitignore

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**